mantis-runtime/src/main/java/io/mantisrx/runtime/GroupToGroup.java

```diff
@@ -23,7 +23,15 @@
 import java.util.List;
 
 
-public class GroupToGroup<K1, T, K2, R> extends StageConfig<T, R> {
+/**
+ * Use to perform another shuffle on the data with a different key
+ * It transforms an item of type MantisGroup<K1, T> to MantisGroup<K2,R>
+ * @param <K1> Input key type
+ * @param <T> Input value type
+ * @param <K2> Output key type
+ * @param <R> Output value type
+ */
```

Comment:
Great job in documenting the parametric types!

Reply:
+1